### DISCUSSION TITLE
Redacting 'target' attribiute from HTTP span?

### DISCUSSION THREAD

**Discussion by vwitteki** [open-telemetry/opentelemetry-ruby#1488] [Q&A]
Hello!  The behavior I am seeking is similar to the `untraced_hosts` configuration of `OpenTelemetry::Instrumentation::Net::HTTP`; but I would like to only remove the target attribute for a specific host, not the whole span.

Not sure if there is a way to do this with the existing config options, or a way to inject middleware to do this?

If not, happy to create a PR :)

**Comment by arielvalentin** [comment]
@vwitteki sorry for the Delayed Response! 

The SDK does not offer out of the box filtering for attributes and neither do the individual instrumentations since it is not part of the specification.

Is there a specific use case that you are trying to address?

Are you using the OpenTelemetry Collector as part of your infrastructure? That is the best way to filter/transform attributes at the moment.
